src/streamlink_cli/argparser.py

```diff
@@ -1070,7 +1070,8 @@ def build_parser():
         "--http-proxy",
         metavar="HTTP_PROXY",
         help="""
-        A HTTP proxy to use for all HTTP requests.
+        A HTTP proxy to use for all HTTP request, by default this proxy
```

Comment:
shouldn't this still be `requests` not `request`

Reply:
Yep